docs/how-to/3rd-party/tensorflow-install.rst

```diff
@@ -72,21 +54,51 @@ Follow these steps:
 Using a wheels package
 -------------------------------------------------------------------------------
 
-To install TensorFlow using the wheels package, use the following command.
+#. Ensure your environment is using a supported Python version for the desired ``tensorflow-rocm`` wheel.
 
-.. code-block:: shell
+   a. To check compatible Python versions, navigate to the desired ROCm release version
```

Comment:
These instructions only apply to ROCm 6.1+ because manylinux only has wheels packages for ROCm 6.1+.  We should highlight that these are only for 6.1+

Reply:
I think needing users to go through the repository to read the filenames isn't a great experience, could we have another table with the supported versions?